server/index-node/src/schema.graphql

```diff
@@ -57,6 +59,21 @@ type EthereumIndexingStatus implements ChainIndexingStatus {
   lastHealthyBlock: Block
 }
 
+type EntityChanges {
+  updates: [EntityData!]!
+  deletes: [String!]!
```

Comment:
For `deletes` you need to report both the entity type and the entity id.

Reply:
Done, see https://github.com/graphprotocol/graph-node/pull/3275#discussion_r816660558. Marking as resolved.